pyaedt/modeler/Modeler.py

```diff
@@ -866,6 +866,7 @@ def add_workbench_link(self, objects, ambient_temp=22, create_project_var=False,
         vdef = []
         for obj in objects:
             mat = self.primitives[obj].material_name
+            print("Material" + mat)
```

Comment:
Let's avoid adding print statements and instead use the standard python logging module.  Users won't want their stdout polluted.

Reply:
Fair enough ...

Reply:
I'll address this in a complete way in a coming PR. Requesting to leave this for now. I'll open an issue to keep track of this